appveyor.yml

```diff
@@ -80,4 +75,18 @@ build_script:
 #---------------------------------#
 
 # disable automatic tests
-test: off
+
+before_test:
+  - ps: cd C:\projects\postgres\src\tools\msvc
+  - install C:\projects\pgsql
+  - cd C:\projects\postgres\contrib\citus\src\backend\distributed
+  - copy citus--*.sql C:\projects\pgsql\share\extension\
+  - perl install.pl C:\projects\pgsql
+
+test_script:
+  - ps: cd C:\projects\postgres\contrib\citus\src\test\regress
+  - perl pg_regress_multi.pl --bindir=C:\projects\pgsql\bin --load-extension=citus -- --schedule=multi_schedule
```

Comment:
Are we planning to run other tests (isolation, mx, ...) in windows?

Reply:
I think that makes sense to do eventually, but the regular regression tests are good enough for now.